### PR TITLE
Bug 1815609 - changes to Acehnese locale name

### PR DESCRIPTION
**Pull request by Delphine**
Also sorted list alphabetically
https://bugzilla.mozilla.org/show_bug.cgi?id=1815609
https://bugzilla.mozilla.org/show_bug.cgi?id=1815609
https://bugzilla.mozilla.org/show_bug.cgi?id=1815609

### CODE REVIEW

**Comment by Delphine** [comment]
Hi @Amejia481, can we land this? It's been open for a couple weeks now. thanks!

**Comment by Amejia481** [comment]
I addressed the lint issues and marked it to land

**Review by Amejia481 — APPROVED** [review]

**Comment by Delphine** [comment]
Thanks for catching and fixing that! 😃

File: focus-android/app/src/main/java/org/mozilla/focus/locale/screen/LocaleDescriptor.kt [hunk 1]
```diff
@@ -28,34 +28,34 @@ class LocaleDescriptor(private val localeTag: String) : Comparable<LocaleDescrip
...
+        languageCodeAndNameMap["trs"] = "Triqui"
+        languageCodeAndNameMap["tsz"] = "P'urhepecha"
+        languageCodeAndNameMap["tt"] = "татарча"
+        languageCodeAndNameMap["wo"] = "Wolof"
+        languageCodeAndNameMap["zam"] = "DíɁztè"
         languageCodeAndNameMap["zh-CN"] = "中文 (中国大陆)"
-        languageCodeAndNameMap["nv"] = "Navajo"
-        languageCodeAndNameMap["skr"] = "سرائیکی"
     }
-
```
Comment (gabrielluong) [inline focus-android/app/src/main/java/org/mozilla/focus/locale/screen/LocaleDescriptor.kt:58]: We probably don't want to remove this new line that separates the function

Comment (Amejia481) [reply focus-android/app/src/main/java/org/mozilla/focus/locale/screen/LocaleDescriptor.kt:58]: 👍🏽

**Comment by Delphine** [comment]
Hi, could we please land this? I think it's fallen through the cracks

**Pull request merged**